src/plugins/mdraid.c

```diff
@@ -402,6 +402,88 @@ gboolean bd_md_create (gchar *device_name, gchar *level, gchar **disks, guint64
 }
 
 /**
+ * bd_md_create_with_chunk_size:
+ * @device_name: name of the device to create
+ * @level: RAID level (as understood by mdadm, see mdadm(8))
+ * @disks: (array zero-terminated=1): disks to use for the new RAID (including spares)
+ * @spares: number of spare devices
+ * @version: (allow-none): metadata version
+ * @bitmap: whether to create an internal bitmap on the device or not
+ * @chunk_size: chunk size of the device to create
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether the new MD RAID device @device_name was successfully created or not
+ */
+gboolean bd_md_create_with_chunk_size (gchar *device_name, gchar *level, gchar **disks, guint64 spares, gchar *version, gboolean bitmap, guint64 chunk_size, GError **error) {
+    gchar **argv = NULL;
+    /* ["mdadm", "create", device, "--run", "level", "raid-devices",...] */
```

Comment:
Curly brackets would be better here as that's how static arrays are defined in C.